### DISCUSSION TITLE
Welcome to wg-modernization Discussions!

### DISCUSSION THREAD

**Discussion by ijmitch** [openmainframeproject/wg-modernization#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MANISH-K-07** [comment]
Hey @ijmitch , @MistyDecker ,
I am Manish K, an undergrad sophomore pursuing Computer Science Engineering in India and keen on open source.
Found the goal of this project monumentally interesting while I was exploring through LFX Mentorships. I would like to help you guys achieve this purpose starting this summer.

I have a good understanding of **Mainframe**, the What, Why and How of **Modernization** and my thoughts seem to align with the goal of the **Modernization Working Group**. 
My consistency in Open source and experience is evident at my [GitHub](https://github.com/MANISH-K-07?tab=overview&from=2024-05-01&to=2024-05-17) page and I have excellent analytical abilities and 
communication skills (also [certified](https://www.credly.com/badges/3114470a-0f55-44f0-8009-f53eea557275) by Pearson). To add, I have profound interest in the field of technical research and have published a paper related to Blockchain technology titled [Charity with clarity: Crowdfunding using Smart contracts](https://github.com/MANISH-K-07/IEEE-Research-Papers/blob/main/Blockchain-Project-IEEE.pdf) at **IEEE**.

I would like to participate this summer as a [mentee](https://mentorship.lfx.linuxfoundation.org/project/3113bbee-4d1b-4e84-92bf-7f78324972bd) under your guidance and learn more about mainframe modernization, research and analyze varying perspectives related to this "transformation" and write a paper that will help many organizations now and in near future.

Please let me know on how I can showcase my skills and prove my experience here...

Thanks,
Manish K ([LinkedIn](https://www.linkedin.com/in/manish-k-kandrakota/))
